extensions/stackdriver/log/exporter.cc

```diff
@@ -56,6 +56,7 @@ ExporterImpl::ExporterImpl(RootContext* root_context,
 
   // Construct grpc_service for the Stackdriver gRPC call.
   GrpcService grpc_service;
+  grpc_service.mutable_google_grpc()->set_stat_prefix("stackdriver_logging");
   if (logging_service_endpoint.empty()) {
     grpc_service.mutable_google_grpc()->set_target_uri(
         kGoogleStackdriverLoggingAddress);
```

Comment:
Can you verify that setting Gooogle_application_credentials env var is honored here?
That is how we will use it on a VM.

Reply:
Yes it does. grpc-cpp library gets it from that env variable. That is how gke gets credentials as well.